src/components/photoGallery/LargeImage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+interface LargeImageProps {
+  largeImageUrl: string;
```

Comment:
Will need to take the alt text
```suggestion
  largeImageUrl: string;
  alt: string;
```